Remove deprecated consent

NSeydoux opened the pull request:
Removing deprecated functions whose name rely on the "consent" terminology, which we are walking away from.

NSeydoux committed:
Remove approveAccessRequestWithConsent

NSeydoux committed:
Remove getAccessWithConsent

NSeydoux committed:
Remove getConsentApiEndpoint

NSeydoux committed:
Remove getConsentManagementUi

NSeydoux committed:
Remove redirectToConsentManagementUi

NSeydoux committed:
Remove requestAccess

NSeydoux committed:
Remove revokeAccess

NSeydoux committed:
Remove isValidConsentGrant

matthieubosquet reviewed (approved):
This looks good, but there are a lot of places in the code and comments, including, most importantly in the types (see ConsentApiBaseOptions and consentEndpoint).

Which should also be removed.

NSeydoux committed:
Remove ConsentApiBaseOptions

NSeydoux committed:
Remove consentEndpoint

NSeydoux committed:
Change ConsentContext into AccessGrantContext

NSeydoux committed:
Remove "consent" from exampl code

NSeydoux committed:
Remove fallbackConsentUi

NSeydoux committed:
Replace CONSENT_CONTEXT with ACCESS_GRANT_CONTEXT

NSeydoux committed:
Merge internal Consent and Access

Some internal code was still making a difference between consent and access based on the presence of a purpose, this is now simplified into just access with an optional purpose.

NSeydoux committed:
Change Consent to GConsent where relevant

To remove any ambiguity, any place where Consent should still be referenced has been changed to GConsent, the RDF vocabulary for consent we use.

NSeydoux committed:
Fix linting

NSeydoux committed:
Fix rollup config
